npm/react/src/createMount.ts

```diff
@@ -109,14 +96,14 @@ export const makeMountFn = (
     }
 
     return (
-      // Separate alias and returned value. Alias returns the component only, and the thenable returns the additional functions
       cy.wrap<React.ReactNode>(userComponent, { log: false })
-      .as(displayName)
```

Comment:
We are removing the alias, we should also remove the type for it in `src/types.ts`

Reply:
Done

Reply:
Revisiting this, we should throw an error if the user passed in `cy.mount(<MyComponent />, { alias: 'MyComponent' })`

Reply:
https://github.com/cypress-io/cypress/pull/24419/commits/e4b9943d4e98bfdb2312ae80902b126ed57255cd